.github/config/file-filters.yml

```diff
@@ -1,8 +1,6 @@
 changed:
   - '.github/workflows/PR-build.yml'
   - 'cmd/**'
-  - 'config/**'
```

Comment:
IMO I should leave the config as the way it is, since someone [could change the config in the future develop](https://github.com/aws-observability/aws-otel-collector/blob/d2fcd75cc8654b08c04705de53955ee5be11f566/cmd/awscollector/Dockerfile#L69)

Reply:
It doesn't look like we build the image as part of the PR-build though.

Reply:
Just kidding. It builds the image as part of the test-cases.